server/service/src/sync/program_schemas/hiv_care_encounter.json

```diff
@@ -72,44 +75,49 @@
               },
               "type": "array"
             },
+            "quantityDispensed": {
+              "description": "Number of pills dispensed in this encounter",
+              "minimum": 0,
+              "type": "number"
+            },
             "quantityPrescribed": {
               "description": "Number of pills on hand at the end of the encounter.",
               "type": "number"
             },
             "regimen": {
               "description": "Antiretroviral treatment regimen",
               "enum": [
-                "1a-New",
-                "1a-Old",
-                "1b-New",
-                "1b-Old",
-                "2a-New",
-                "2a-Old",
-                "2b-New",
-                "2b-Old",
-                "2c-New",
-                "3a-New",
-                "3a-Old",
-                "3b-New",
-                "3b-Old",
-                "3c-New",
-                "4a-New",
-                "4a-Old",
-                "4b-New",
-                "4b-Old",
-                "4c-New",
-                "4c-Old",
-                "5a-New",
-                "5a-Old",
-                "5b-New",
-                "5b-Old",
-                "6a-New",
-                "6a-Old",
-                "6b-New",
-                "6b-Old",
-                "6c-New",
-                "6c-Old",
-                "Other"
+                "1a-New TDF + 3TC + DTG",
```

Comment:
Sorry, I should have had a look at this issue earlier. There is already a "show" option for the `Select` component that can be used to display a description, e.g. `regimenChangeReason`. Or where you aware of it and deliberately choose this option?

Using shorter identifier would make the decision tree definition easier but not sure if it would be ok to show the short version in the "Current ARV Regimen"....